website/docs/cli/commands/auth/tutorials/configuring-geodesic.mdx

```diff
@@ -0,0 +1,255 @@
+
+# Path to the teams config file for assume-role and other Geodesic utilities.
+# Once logged in with Atmos, this config file gives you access to all the other teams
+# and roles (if you are authorized for access).
+ENV AWS_CONFIG_TEAMS=/etc/aws-config/aws-config-teams
+ENV ASSUME_ROLE_INTERACTIVE_QUERY=${NAMESPACE}${TENANT:+-$TENANT}-gbl-
+```
+
+**Key Configuration Details:**
+
+- **`ATMOS_AUTH_PROVIDER`**: Name of the provider defined in your `atmos.yaml` (e.g., `acme-sso`) - used to construct credential file paths
+- **`ATMOS_AUTH_IDENTITY`**: Name of the identity to use by default (must match an identity in `atmos.yaml`)
+- **`AWS_SHARED_CREDENTIALS_FILE`**: Points to Atmos-managed credentials directory
+- **`AWS_CONFIG_FILE`**: Points to Atmos-managed AWS config file
+- **`AWS_PROFILE`**: The AWS profile name (set to match identity)
+- **`ATMOS_IDENTITY`**: Environment variable that Atmos reads to determine default identity
+- **`AWS_CONFIG_TEAMS`**: Separate variable for teams config (avoids clobbering `AWS_CONFIG_FILE`)
+
+:::note
+The `AWS_CONFIG_FILE` environment variable must remain pointed at the Atmos-managed config (`$HOME/.aws/atmos/${ATMOS_AUTH_PROVIDER}/config`) for authentication to work correctly. The teams config is stored in the separate `AWS_CONFIG_TEAMS` variable. If your Geodesic utilities reference the teams config, update them to use `AWS_CONFIG_TEAMS` instead of `AWS_CONFIG_FILE`.
+:::
+
+### 2. Update Your Makefile
+
+Add an automatic login step to your Makefile to ensure users are authenticated before starting the Geodesic shell:
+
+```makefile
+login:
+	@atmos auth login --identity acme-identity
+
+...
+
+## Start the geodesic shell by calling wrapper script
+run: login
+	@$(APP_NAME)
+```
+
+This ensures that:
+- Users authenticate with Atmos before the shell starts
+- The specified identity is used for authentication
+- Credentials are fresh when starting work
+
+### 3. Update Source Profile Configuration
+
+For `assume-role` and other Geodesic utilities to work correctly, update your `source_profile` in the `{Repo Root}/etc/aws-config/aws-config-teams` file to match the identity name from your `atmos.yaml` auth configuration.
+
+**Example:**
+
+If your `atmos.yaml` defines:
+```yaml
+auth:
+  identities:
+    acme-identity:
+      kind: aws/permission-set
+      # ...
+```
+
+Then your `aws-config-teams` should reference:
+```ini
+[profile some-role]
+source_profile = acme-identity
+role_arn = arn:aws:iam::123456789012:role/SomeRole
+```
+
+## Complete Example
+
+Here's a complete example showing how all the pieces fit together:
+
+### atmos.yaml
+```yaml
+auth:
+  providers:
+    acme-sso:
+      kind: aws/iam-identity-center
+      region: us-east-1
+      start_url: https://acme.awsapps.com/start/
+
+  identities:
+    acme-identity:
+      default: true
+      kind: aws/permission-set
+      via:
+        provider: acme-sso
+      principal:
+        name: "AdminAccess"
```

Comment:
For refarch geodesic / leapp migration we should show the actual permission set they'd need to set

```suggestion
        name: "IdentityDevopsTeamAccess"
```